pkg/manager/impl/testutils/repository.go

```diff
@@ -13,7 +14,8 @@ func GetRepoWithDefaultProjectAndErr(err error) repositories.RepositoryInterface
 	repo := repositoryMocks.NewMockRepository()
 	repo.ProjectRepo().(*repositoryMocks.MockProjectRepo).GetFunction = func(
 		ctx context.Context, projectID string) (models.Project, error) {
-		return models.Project{}, err
+		activeState := int32(admin.Project_ACTIVE)
+		return models.Project{State: &activeState}, err
```

Comment:
I'm not so sure here. Do we usually assume migration will always happen so `state` will always exist? I have to do this because point deference `if *project.State != int32(admin.Project_ACTIVE)`

Reply:
we usually run the migrations script as part of the admin initcontainers so I think it is safe to assume you can dereference. this test data anyways, so it should use the latest model definition :)